cypress/support/commands.js

```diff
@@ -0,0 +1,5 @@
+/// <reference types="cypress" />
+
+Cypress.Commands.add("dataCy", (value) => {
```

Comment:
I haven't done this with any of my projects yet! Very cool, you've inspired me to try it out!